internal/generator/binding/object.go

```diff
@@ -80,6 +80,13 @@ func (object *Object) ProcessAnnotations(a map[string]*Annotation) error {
 			return fmt.Errorf("sync annotation value must be empty")
 		}
 		object.ModelEntity.AddFlag(model.EntityFlagSyncEnabled)
+
+		if a["sync"].Details != nil && a["sync"].Details["sharedGlobalIds"] != nil {
```

Comment:
Maybe change the docs of `Details` in `annotation.go`? Currently talking about to-many-relations which threw me off a little.

Reply:
done